### PR TITLE
datavolume_test: tar archive import

### PR DESCRIPTION
**Pull request by danielerez**
Added positive and negative functional tests for importing tar archive files.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/datavolume_test.go [hunk 1]
```diff
@@ -50,7 +51,7 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 		}
 	})
 
-	Describe("Verify DataVolume", func() {
+	FDescribe("Verify DataVolume", func() {
```
Comment (awels) [inline tests/datavolume_test.go]: Remove the F

File: tests/datavolume_test.go [hunk 2]
```diff
@@ -122,6 +125,8 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			table.Entry("[rfe_id:138][crit:high][test_id:1362]succeed creating upload dv", "upload", "", "", "upload-dv", controller.UploadReady, cdiv1.Succeeded),
 			table.Entry("[rfe_id:1115][crit:high][test_id:1478]succeed creating import dv with given valid registry url", "import-registry", "", tinyCoreIsoRegistryURL, "dv-phase-test-4", controller.ImportSucceeded, cdiv1.Succeeded),
 			table.Entry("[rfe_id:1115][crit:high][test_id:1379]succeed creating import dv with given valid url (https)", "import-https", "", httpsTinyCoreIsoURL, "dv-phase-test-1", controller.ImportSucceeded, cdiv1.Succeeded),
+			table.FEntry("[rfe_id:1947][crit:high][test_id:2145]succeed creating import dv with given tar archive url", "import-archive", "", tarArchiveURL, "tar-archive-dv", controller.ImportSucceeded, cdiv1.Succeeded),
```
Comment (awels) [inline tests/datavolume_test.go]: Remove the F

File: tests/datavolume_test.go [hunk 3]
```diff
@@ -122,6 +125,8 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
...
+			table.FEntry("[rfe_id:1947][crit:high][test_id:2220]fail creating import dv with non tar archive url", "import-archive", "", tinyCoreIsoURL, "non-tar-archive-dv", controller.ImportFailed, cdiv1.Failed),
```
Comment (awels) [inline tests/datavolume_test.go]: Remove the F

File: tests/datavolume_test.go (hunk 1 above)
Comment (danielerez) [reply tests/datavolume_test.go]: Right, just noticed it :) removed.

File: tests/datavolume_test.go (hunk 2 above)
Comment (danielerez) [reply tests/datavolume_test.go]: Done.

**Comment by awels** [comment]
Looks like the linter doesn't like this, make test-lint will let you know what is wrong.
https://travis-ci.org/kubevirt/containerized-data-importer/builds/541737275#L5714

**Pull request merged**